From: Steve Sakoman
Subject: omap3-sgx-modules: Fix build for Linux 3.0

This patch implements the header change from plat/display.h to video/omapdss.h

Signed-off-by: Steve Sakoman <steve@sakoman.com>

--- GFX_Linux_KM/services4/3rdparty/dc_omap3430_linux/omaplfb_linux.c-orig	2011-08-01 07:21:45.000000000 -0700
+++ GFX_Linux_KM/services4/3rdparty/dc_omap3430_linux/omaplfb_linux.c	2011-08-01 07:29:32.000000000 -0700
@@ -47,12 +47,12 @@
 #if defined (SUPPORT_TI_DSS_FW)
 #include <asm/io.h>
 
-#if (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,26))
-#if (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,31))
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,39)))
+#include <video/omapdss.h>
+#elif (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,31))
 #include <plat/display.h>
-#else 
+#elif (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,26))
 #include <mach/display.h>
-#endif 
 #else 
 #include <asm/arch-omap/display.h>
 #endif 
@@ -64,7 +64,11 @@ extern int omap_dispc_request_irq(unsign
 extern void omap_dispc_free_irq(unsigned long, void (*)(void *), void *);
 extern void omap_dispc_set_plane_base(int plane, IMG_UINT32 phys_addr);
 #else
+#if (LINUX_VERSION_CODE > KERNEL_VERSION(2,6,39))
+#include <video/omapdss.h>
+#else
 #include <plat/display.h>
+#endif
 #include <linux/console.h>
 #include <linux/fb.h>
 static omap_dispc_isr_t *pOMAPLFBVSyncISRHandle = NULL;